orchestrator/orchestrator.go

```diff
@@ -365,13 +411,13 @@ func (s *Orchestrator) startExecutionStream(ctx context.Context) error {
 					attr := event.Events[attrKeyHash][index]
 					hash, err := hash.Decode(attr)
 					if err != nil {
-						s.ErrC <- err
+						s.logger.Error(err.Error(), err)
```

Comment:
Again here why not pushing to the pubsub? and the 2 other bellow

Reply:
same as https://github.com/mesg-foundation/engine/pull/1812#discussion_r422084118